api.go

```diff
@@ -81,3 +100,38 @@ func (a *API) RouteTree() string {
 
 	return routes
 }
+
+// SendAndLog is a jsh wrapper function that first prepares a jsh.Sendable response,
+// and then handles logging 5XX errors that it encounters in the process.
+func (a *API) SendAndLog(ctx context.Context, w http.ResponseWriter, r *http.Request, sendable jsh.Sendable) {
+	if a == nil {
```

Comment:
This case could be encountered if a handler on a Resource is called without adding the Resource to an API. That's not the intended use of the type, but  I guarded against it to maintain backward compatibility because Resource is an exported type.